nevergrad/benchmark/experiments.py

```diff
@@ -72,8 +72,7 @@ def yawidebbob(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     # Discrete, unordered.
     for nv in [10, 50, 200]:
         for arity in [2, 7]:
-            variables = list(ng.p.TransitionChoice(list(range(arity))) for _ in range(nv))
-            instrum = ng.p.Instrumentation(*variables)
+            instrum = ng.p.Tuple(*(ng.p.TransitionChoice(range(arity)) for _ in range(nv)))
```

Comment:
@teytaud What you wanted is to have 1 argument which was a tuple of the integer choices, so you should just build it. The `Instrumentation` class is for when you want multiple arguments (positional and/or keyword arguments). So this is not what you needed here.

Reply:
Ok thx!